docs/docs/components/pressable.mdx

```diff
@@ -0,0 +1,154 @@
+---
+id: pressable
+title: Pressable
+sidebar_label: Pressable
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import GifGallery from '@site/components/GifGallery';
+
+<GifGallery>
+  <img src={useBaseUrl('gifs/pressable.gif')} width="70%" />
+</GifGallery>
+
+:::info
+This component is a drop-in replacement for the `Pressable` component.
+:::
+
+`Pressable` is a component that can detect various stages of tap, press and hover interactions on any of its defined children.
+
+### Usage:
+
+To use `Pressable`, import it in the following way:
+
+```js
+import { Pressable } from 'react-native-gesture-handler';
+```
+
+## Properties
+
+### `children`
+
+either children or a render prop that receives a boolean reflecting whether
+the component is currently pressed.
+
+### `style`
+
+either view styles or a function that receives a boolean reflecting whether
+the component is currently pressed and returns view styles.
+
+### `onPress`
+
+called after `onPressOut` when a single tap gesture is detected.
+
+### `onPressIn`
+
+called before `onPress` when a touch is engaged.
+
+### `onPressOut`
+
+called before `onPress` when a touch is released.
+
+### `onLongPress`
+
+called immidiately after pointer has been down for at least `delayLongPress` milliseconds (`500` ms by default).
+
+after `onLongPress` has been called, `onPressOut` will be called as soon as the pointer is lifted and `onPress` will no longer be called at all.
+
+### `cancelable`
+
+whether a press gesture can be interrupted by a parent gesture such as a
+scroll event. Defaults to `true`.
+
+### `onHoverIn` (Web only)
+
+called when pointer is hovering over the element.
+
+### `onHoverOut` (Web only)
+
+called when pointer stops hovering over the element.
+
+### `delayHoverIn` (Web only)
+
+duration to wait after hover in before calling `onHoverIn`.
+
+### `delayHoverOut` (Web only)
+
+duration to wait after hover out before calling `onHoverOut`.
+
+### `delayLongPress`
+
+duration (in milliseconds) from `onPressIn` before `onLongPress` is called.
+
+### `disabled`
+
+whether the `Pressable` behavior is disabled.
+
+### `hitSlop`
+
+additional distance outside of this view in which a press is detected and `onPressIn` is triggered.
```

Comment:
Would be nice to mention what is the type of this prop. It reads as if only numbers are supported, but you can also pass a `Rect` here.

Reply:
Which one do you prefer?
![image](https://github.com/user-attachments/assets/0a9d1933-3a6a-4c7c-a2f6-07898e1f56b6)
![image](https://github.com/user-attachments/assets/ef190800-7310-4b35-9db5-35331d86c98f)

I think the first one makes it easier to see that `Rect` is a clickable link.

Reply:
+1 for the second one